models/pull.go

```diff
@@ -66,6 +66,7 @@ type PullRequest struct {
 	HeadBranch   string
 	BaseBranch   string
 	MergeBase    string `xorm:"VARCHAR(40)"`
+	LastCommitID string `xorm:"VARCHAR(40)"`
```

Comment:
Why do we need `LastCommitID`? It is only used once in assignment.

Reply:
@daviian #2519 will need it.

Reply:
It will help show the pull's commit status. If we didn't cache it, we will spent many time to invoke `git` command on that PR.

Reply:
Ok thanks for clarification 👍